src/App.tsx

```diff
@@ -15,36 +15,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Alphabetically = 'alphabetically',
+  ByLength = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  reversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      if (sortField === SortType.Alphabetically) {
+        return good1.localeCompare(good2);
+      }
+
+      if (sortField === SortType.ByLength) {
+        return good1.length - good2.length;
+      }
+
+      return 0;
+    });
+  }
+
+  if (reversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState(SortType.None);
+  const [reversed, setReversed] = useState(false);
+  const isSorted = sortField !== '';
```

Comment:
The `isSorted` logic currently checks if `sortField` is not an empty string. However, it should also consider the `reversed` state to accurately reflect whether the goods are sorted. Consider revising this logic to account for both `sortField` and `reversed`.